src/stories/containers/auth/change-password/change-password.tsx

```diff
@@ -1,17 +1,23 @@
 import styled from '@emotion/styled';
-import React from 'react';
+import React, { useCallback } from 'react';
 import CloseButton from '../../../components/close-button/close-button';
 import { useThemeContext } from '../../../../core/context/ThemeContext';
 import { CustomButton } from '../../../components/custom-button/custom-button';
 import TextInput from '../../../components/text-input/text-input';
 import { ButtonWrapper, Container, Form, Wrapper } from '../login/login';
 import { userChangePasswordMvvm } from './change-password.mvvm';
 import { CircleAvatar } from '../../../components/circle-avatar/circle-avatar';
+import { useRouter } from 'next/router';
 
 export default () => {
   const { isLight } = useThemeContext();
+  const router = useRouter();
   const { form, username, loading, error } = userChangePasswordMvvm();
 
+  const handleGoBack = useCallback(() => {
+    router.back();
```

Comment:
When the user goes directly to the URL, the "back" status will be not available so it will no able to go back. We can just redirect to the profile page maybe or check if the router has the possibility to go back and if it hasn't, just do the redirect